packages/api-client/src/models/VMs.ts

```diff
@@ -176,11 +180,11 @@ export function VMsFromJSONTyped(json: any, ignoreDiscriminator: boolean): VMs {
         'diskCount': VMResourceBreakdownFromJSON(json['diskCount']),
         'nicCount': json['nicCount'] == null ? undefined : VMResourceBreakdownFromJSON(json['nicCount']),
         'powerStates': json['powerStates'],
-        'os': json['os'],
+        'os': json['os'] == null ? undefined : json['os'],
         'osInfo': json['osInfo'] == null ? undefined : (mapValues(json['osInfo'], OsInfoFromJSON)),
-        'notMigratableReasons': json['notMigratableReasons'] == null ? undefined : ((json['notMigratableReasons'] as Array<any>).map(MigrationIssueFromJSON)),
```

Comment:
@Jos-Jerus please don't remove these validations for arrays. If we receive some null values this cause a crash error in the UI